doc/hooks/waf.section.md

```diff
@@ -1,50 +1,58 @@
-# waf.hook {#wafhook}
+# wafHook {#waf-hook}
 
 [Waf](https://waf.io) is a Python-based software building system.
 
-In Nixpkgs, `waf.hook` overrides the default configure, build, and install phases.
+In Nixpkgs, `wafHook` overrides the default configure, build, and install phases.
 
-## Variables controlling waf.hook {#variablesControllingWafHook}
+## Variables controlling wafHook {#waf-hook-variables-controlling}
 
-### `wafPath` {#wafPath}
+### `wafHook` Exclusive Variables {#waf-hook-exclusive-variables}
 
-Location of the `waf` tool. It defaults to `./waf`, to honor software projects that include it directly inside their source trees.
+The variables below are exclusive of `wafHook`.
```

Comment:
these changes should probably be reflected in the commit message

Reply:
I could swear that I did this.